facia-purger/README.md

```diff
@@ -0,0 +1,9 @@
+# facia-purger
+
+facia-purger is a lambda for listening to events from the fapi/pressed S3 objects in order to purge fastly when an object is changed
+
+## Tests
+
+Note that in order to run the included integration test you will need to have valid AWS credentials configured in `.aws/credentials`.
+
+By default, the "default" profile will be used - in order to use a different profile (most likely frontend) set the `AWS_PROFILE` environment variable.
```

Comment:
Agreed, it wasn't immediately easy to spot the best way to do this with the Java SDK, but I'll take another look.